iothub/service/src/ServiceClient.cs

```diff
@@ -29,7 +29,8 @@ public enum TransportType
 #pragma warning restore CA1707 // Identifiers should not contain underscores
 
     /// <summary>
-    /// Contains methods that services can use to send messages to devices
+    /// Contains methods that services can use to send messages to devices/ modules,
+    /// invoke a direct method on a device/ module and receive file upload and cloud to device message delivery notifications.
```

Comment:
This reads like a run-on sentence until one realizes that file upload and C2D are a compound adjective for notifications. I'd reword it to be like "and delivery notifications for file upload and cloud-to-device operations."

Reply:
cloud-to-device with hyphens?

Reply:
yes because it is a compound adjective